src/v/serde/serde_is_enum.h

```diff
@@ -17,7 +17,7 @@ using serde_enum_serialized_t = int32_t;
 
 template<typename T>
 inline constexpr bool serde_is_enum_v =
-#if __has_cpp_attribute(__cpp_lib_is_scoped_enum)
+#if __cpp_lib_is_scoped_enum >= 202011L
```

Comment:
What changed that made this break?

Reply:
I've reverted this change now. It was needed because seastar was overwriting the c++ version used to 23 instead of 20, and it looks like this fails in c++ 23.